### PR TITLE
Archeo: Update all text in patterns

### LINKED ISSUE #5484: Archeo: Update all text in patterns
Once all patterns (https://github.com/Automattic/themes/issues/5429) have been merged, replace all the text with English and ensure the text is consistently matched to the images across all patterns.

### PR DESCRIPTION
**Pull request by mikachan**
#### Changes proposed in this Pull Request:
This updates all text in the Archeo patterns to English text, according to @beafialho's updates [here](https://github.com/Automattic/themes/issues/5484#issuecomment-1043164119).

#### Related issue(s):
Closes https://github.com/Automattic/themes/issues/5484

### CODE REVIEW

File: archeo/inc/patterns/layered-images-with-headline.php [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
 	<div class="wp-block-columns are-vertically-aligned-top" style="padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:column {"verticalAlignment":"top"} -->
 	<div class="wp-block-column is-vertically-aligned-top"><!-- wp:spacer {"height":"14vw"} -->
 	<div style="height:14vw" aria-hidden="true" class="wp-block-spacer"></div>
 	<!-- /wp:spacer -->
 	
 	<!-- wp:paragraph {"style":{"typography":{"textTransform":"uppercase","fontStyle":"normal","fontWeight":"100","lineHeight":"1"}}} -->
-	<p style="font-size:clamp(64px, 6vw, 100px);font-style:normal;font-weight:100;line-height:1;text-transform:uppercase">' . wp_kses_post( __( 'Palais Du <br>Cirque, à Chichen-Itza, bas-relief<br>des tigres.', 'archeo' ) ) . '</p>
+	<p style="font-size:clamp(64px, 6vw, 100px);font-style:normal;font-weight:100;line-height:1;text-transform:uppercase">' . wp_kses_post( __( 'Palace of <br>the Circus at <br>Chichen-Itza, <br>bas-relief of <br>tigers', 'archeo' ) ) . '</p>
```
Comment (jffng) [inline archeo/inc/patterns/layered-images-with-headline.php]: Is _BAS-RELIEF_ supposed to be italicized? 

Pattern-8

**Review by jffng — COMMENTED** [review]
One small comment, otherwise LGTM.

File: archeo/inc/patterns/layered-images-with-headline.php (hunk 1 above)
Comment (kjellr) [reply archeo/inc/patterns/layered-images-with-headline.php]: I don't think it needs to be.

Comment (mikachan) [reply archeo/inc/patterns/layered-images-with-headline.php]: Good spot! Just fixed it.

Comment (beafialho) [reply archeo/inc/patterns/layered-images-with-headline.php]: That's right, bas-relief should be in italic 👍

**Review by jffng — APPROVED** [review]

**Pull request merged**